docs/dev/development.adoc

```diff
@@ -333,7 +340,7 @@ There are some test environment variable that helps to get more control over the
 
 *Running integration tests:*
 
-By default, tests are run against the `odo` binary placed in the PATH which is created by command `make`. Integration tests can be run in two (parallel and sequential) ways. To control the parallel run use environment variable `TEST_EXEC_NODES`. For example component test can be run
+For running tests on a 3.11 cluster, login to the cluster using the required credentials. For example `odo login -u <user_name> -p <password> <server_URL>`. However `make configure-installer-tests-cluster` performs the login operation required to run the tests on a 4.* cluster. By default, tests are run against the `odo` binary placed in the PATH which is created by the command `make`. Integration tests can be run in two (parallel and sequential) ways. To control the parallel run use the environment variable `TEST_EXEC_NODES`. For example the component test can be run
```

Comment:
```suggestion
For running tests on a 3.11 cluster, login to the cluster using the required credentials. For example `odo login -u <user_name> -p <password> <server_URL>`. In case of 4.* cluster, `make configure-installer-tests-cluster` performs the login operation required to run the test. By default, the tests are run against the `odo` binary placed in the $PATH which is created by the command `make`. Integration tests can be run in two ways, parallel and sequential. To control the parallel run, use the environment variable `TEST_EXEC_NODES`. For example, the component test can be run as following:
```